js-miniapp-sample/.flowconfig

```diff
@@ -0,0 +1,25 @@
+[declarations]
+<PROJECT_ROOT>/../node_modules/.*
+<PROJECT_ROOT>/node_modules/.*
+<PROJECT_ROOT>/src/serviceWorker.js
+<PROJECT_ROOT>/src/setupTests.js
+
+[ignore]
+<PROJECT_ROOT>/build/.*
+<PROJECT_ROOT>/public/.*
+<PROJECT_ROOT>/src/.*.test.js
+<PROJECT_ROOT>/src/serviceWorker.js
+<PROJECT_ROOT>/src/setupTests.js
+[include]
+
+[libs]
+
+[lints]
+
+[options]
+# all=true
```

Comment:
If we comment this line here is flow still going to check all the .js files?

Reply:
updated the flowconfig